python/pyarrow/tests/test_scalars.py

```diff
@@ -295,6 +296,9 @@ def test_cast():
         pa.scalar('foo').cast('int32')
 
 
+# TODO: We should test on windows once ARROW-13168 is resolved.
+@pytest.mark.skipif(sys.platform == 'win32',
+                    reason="Timezone database is not available on Windows yet")
```

Comment:
Perhaps a more correct wording (so as to not confuse the reader) would be:
```suggestion
@pytest.mark.skipif(sys.platform == 'win32',
                    reason="Timezone database is sometimes not available on Windows yet")
```